### PR TITLE
fixes #32: adding getting involved info to README.md

### LINKED ISSUE #32: FAQ: Getting Involved
When someone hears about the Beman Project and wants to get involved, there should be a quick step-by-step to discover could be done, pick up some of that work, and get support when completing that work.

This issue will be done when there is an FAQ entry for getting involved in the Beman Project. If the content becomes too long for an entry for an FAQ, creating a standalone document and linking to it from the FAQ is acceptable.

For reference, [here is a link to the existing FAQ](https://github.com/beman-project/beman#faq) for reference.

**Mentor**: @bretbrownjr is available to answer questions and review pull requests for this issue.

### PR DESCRIPTION
**Pull request by linusboehm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,21 +14,31 @@ We have two principle audiences: Library Developers and the C++ community.  We w
 
 This project is organized by our [Governance structure](docs/governance.md).
 
-### FAQ
+### Participating and contributing
+
+We welcome anyone, who wants to participate in the Beman project either through discussions on
+[Discourse](https://discourse.boost.org/t/welcome-to-beman-project-development/3), or through
+contributions on GitHub. If you are interested, please join our discourse community to stay up to
+date with the most recent developments.
```
Comment (camio) [inline README.md:22]:
```suggestion
We welcome all to participate in the Beman project through 
[Discourse](https://discourse.boost.org/t/welcome-to-beman-project-development/3) discussions or GitHub contributions. Our discourse community is the best way to stay up to
date with the most recent developments.
```

File: README.md [hunk 2]
```diff
@@ -14,21 +14,31 @@ We have two principle audiences: Library Developers and the C++ community.  We w
...
 
-[Discourse for new joiners](https://discourse.boost.org/t/welcome-to-the-beman-project-discourse-start-here/40): start here
+If you are interested to contribute on GitHub, please take a look at the open issues of the
+different [repositories](https://github.com/orgs/beman-project/repositories) of the Beman
+organization. Issues that are marked as `good first issue` are a good starting point for new
+contributors. We try to have an assigned mentor for each of these issues to give guidance to new
+contributors. Don't hesitate to post on discourse, if you have any questions.
```
Comment (camio) [inline README.md:28]:
```suggestion
If you are looking for ways to contribute code, see the open issues of [beman-project repositories](https://github.com/orgs/beman-project/repositories). The [beman repository's issue list](https://github.com/beman-project/beman/issues), in particular, has implementation requests for various papers. Issues marked with `good first issue` are perfect for new
contributors and usually have an assigned mentor. Don't hesitate to post on discourse with any questions.
```

**Review by camio — COMMENTED** [review]
Looks good. I made a few wording tweaks for readability.

File: README.md (hunk 1 above)
Comment (linusboehm) [reply README.md]: thanks, updated!

File: README.md (hunk 2 above)
Comment (linusboehm) [reply README.md]: updated

**Review by camio — APPROVED** [review]

**Comment by camio** [comment]
I approved this. Please let me know if you can't merge it yourself.

**Pull request merged**